RCNN_tf.py

```diff
@@ -474,6 +476,7 @@ def __init__(
 
     def train(self, train_data_feats, train_data_labels, val_data_feats,
               test_data_feats, test_data_labels, val_data_labels,
+              learning_depth=3, test_depth=6,
               batch_size=100, epochs=7, dropout_rate=0.2,
               adaptive_learning_factor=.1, performance_update_threshold=3,
               min_difference_accuracy=.005, create_graph=True, print_vars=True,
```

Comment:
Please describe ```learning_depth``` and ```test_depth``` in the docstring, to keep it consistent and complete.

Reply:
:+1: